doc/whats_new/v0.20.rst

```diff
@@ -419,4 +423,4 @@ Changes to estimator checks
 
 - Add test :func:`estimator_checks.check_methods_subset_invariance` to check
   that estimators methods are invariant if applied to a data subset.
-  :issue:`10420` by :user:`Jonathan Ohayon <Johayon>`
```

Comment:
Try to get rid of this strange diff.

Reply:
I don't know, the file in the master has already that line. Should I remove it?

Reply:
If you actually don't change anything and find it hard to get rid of it, you might just keep it. (Hope there won't be some strange things when merging)